meshtastic/mesh_interface_runtime/request_wait.py

```
@@ -0,0 +1,542 @@
+        Factory that returns the shared acknowledgment state object.
+    get_timeout : Callable[[], Timeout]
+        Factory that returns the shared timeout configuration object.
+    retired_wait_ttl_seconds : float
+        Time-to-live in seconds for retired request IDs before pruning.
+    """
+
+    def __init__(
+        self,
+        *,
+        lock: threading.RLock,
+        get_response_handlers: Callable[[], dict[int, "ResponseHandler"]],
```

Ruff (0.15.7)

[warning] Remove quotes from type annotation

Remove quotes

(UP037)